sdgym/synthesizers/uniform.py

```diff
@@ -24,8 +28,6 @@ def _sample_from_synthesizer(self, synthesizer, n_samples):
             kind = column.dtype.kind
             if kind == 'i':
                 values = np.random.randint(column.min(), column.max() + 1, size=self.length)
-            elif kind == 'O':
-                values = np.random.choice(column.unique(), size=self.length)
```

Comment:
why did we get rid of this?

Reply:
The old code allowed us to not pass any transforms to an object - so would be analogous to doing `update_transformers_by_sdtype('categorical', transformer_name=None)` - which we can't do anymore. This distinction is why I had to make some changes to both synthesizers.

In this case, the `fit` always transforms all columns into numerical, so this line never runs, which is why I deleted it. That's the same reason I removed the check in line 37 for the `IndependentSynthesizer`.